dotnet/src/Connectors/Connectors.Memory.Redis/RedisMemoryStore.cs

```diff
@@ -246,12 +246,19 @@ public async Task RemoveBatchAsync(string collectionName, IEnumerable<string> ke
             cancellationToken: cancellationToken).FirstOrDefaultAsync(cancellationToken: cancellationToken).ConfigureAwait(false);
     }
 
+    /// <summary>
+    /// Dispose the object.
+    /// </summary>
```

Comment:
Can we just inherit this?

Reply:
I don't think so. When I use <inheritcdoc/>, there's no documentation that's shown when I hover over the method.